concepts/access_control.md

```diff
@@ -0,0 +1,72 @@
+| :--------------------------- | :----------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------- | :--------------- |
+| **Attribute Namespace**      | `https://example.com`                                  | Typically a domain controlled by the authoritative source of the attribute definition. Recommended to use a stable, controlled namespace.     | No (by itself)   |
+| **Attribute Canonical Name** | `https://example.com/attr/classification`              | Combination of Namespace and Name (`{Namespace}/attr/{Name}`). **This MUST be globally unique** and identifies the specific attribute *type*. | **Yes**          |
+| **Attribute Instance**       | `https://example.com/attr/classification/value/secret` | The full URI (`{Canonical Name}/value/{Value}`). This represents a specific, actionable attribute assertion used in policies or entitlements. | **Yes**          |
+
+## Attribute Definitions (External Rules)
+
+Crucially, the *rules* governing how attributes are compared are defined in **Attribute Definitions**, which are associated with the globally unique **Attribute Canonical Name**. These definitions are managed by the attribute authority and are stored **outside** the TDF manifest itself.
+
+An Attribute Definition typically includes:
+
+*   **Rule Type:** How multiple values or comparisons should be handled (e.g., `AllOf` - entity must have all specified values, `AnyOf` - entity must have at least one, `Hierarchy` - values have an order).
+*   **Allowed Values:** An optional enumeration or pattern restricting valid attribute values.
+*   **Order/Rank (for Hierarchy):** Defines the relationship between values in a hierarchical attribute (e.g., Confidential < Secret < TopSecret).
+
+The PEP retrieves these definitions at access decision time based on the Canonical Names found in the TDF's `dataAttributes` and the entity's entitlements.
+
+## Access Control Flow in OpenTDF
+
+When an entity requests access to an OpenTDF object:
+
+1.  **Request Initiation:** The client presents the relevant [Key Access Object(s)](../schema/OpenTDF/key_access_object.md) from the TDF manifest to the appropriate KAS, along with the client's credentials and asserted **Entity Entitlements** (Subject Attributes).
+2.  **PEP Evaluation:** The KAS performs the following checks based on the TDF's embedded [Policy Object](../schema/OpenTDF/policy.md) (extracted from the `policy` field):
+    *   **Dissemination Check (if applicable):** If the policy's `dissem` list is present and non-empty, the PEP verifies if the requesting entity's identifier is in the list. If not, access is **denied**.
```

Comment:
more mentions of dissem here, maybe we do want to include it but double checking

Reply:
I believe we want to mention it, but we can run it by @willackerly